docs/book/src/introduction/core_library.md

```diff
@@ -0,0 +1,162 @@
+# Core Library
+
+The Sway Core Library, like the name suggests contains core operators and logic for the primitive types of the Sway programming language. These traits and methods are an extension of the [primitive types](https://docs.fuel.network/docs/sway/basics/built_in_types/#primitive-types) `u8`, `u16`, `u32`, `u64`, `u256`, `str[]`, `str`, `bool` and , `b256` and can be used where appropriate.
+
+> Please note that Sway Core Library at the time of writing (v0.61.0) does not support operators for unsigned integers of size 128 (`u128`). Although the necessary OP codes exists for operators `u128` in the [FuelVM instruction set](https://docs.fuel.network/docs/specs/fuel-vm/instruction-set/) the team has limited bandwidth preparing for mainnet launch, so please consider [contributing](https://docs.fuel.network/docs/sway/reference/contributing_to_sway/) if this is of interest to you.
+
+The latest core library documentation can be found in the [Core Library Book](https://fuellabs.github.io/sway/master/core/). If the latest version is not compatible please refer to the appropriate tagged release.
+
+## Using the Core Library
+
+Unlike their standard library counterparts, core library functionalities do not need to be explicitly imported and will work out of the box after creating any new Sway project with `forc new`. The `use` keyword is simply not required.
+
+Consider this example of using the modulo function for two like value types in a `struct`:
+
+```sway
+let struct1 = MyStruct { val: 10 };
+let struct2 = MyStruct { val: 2 };
+let result_struct = struct1 % struct2;
+```
+
+A bonus of developing with Sway is that developers do not have to worry about underflows and overflows, as these are handled by the FuelVM by design.
```

Comment:
This sentence suggests that Sway and FuelVM are joined at the hip, and while that is true for now, one of the design goals of Sway was to make it platform independent. 

I suggest rephrasing this so that FuelVM is not mentioned, e.g.,

"The developer does not need to worry about underflows and overflows, since the Sway compiler ensures that these are handled at execution time."

In the case of FuelVM the compiler handles underflows and overflows by deferring the checks to the VM, but that's irrelevant for the Sway language itself.